src/action/services/useApi.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import * as FetchApi from "../fetch.js";
+
+function useApi() {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
```js
  const [isLoading, setIsLoading] = useState(true);
  ```
  
  Please explain me why the shit you've done is the definition of stupid.

Reply:
i will politely refuse this request with a little shake with my head
i will just change it and deny it ever happend